service/service_mocks.go

```diff
@@ -0,0 +1,85 @@
+package service
+
+import (
+	"errors"
+	"github.com/Shopify/sarama"
+	"github.com/companieshouse/chs.go/kafka/consumer/cluster"
+	"reflect"
+)
+
+const resourceKind = "resourceKind"
+const resourceUri = "resourceUri"
+const contextID = "contextID"
+const resourceID = "resourceID"
+
+type MockConsumer struct{}
+
+type MockMarshaller struct{}
+
+type MockUpsert struct{}
+
+type MockUpsertFail struct{}
+
+type MockGroup struct{}
```

Comment:
This is a lot of mocking in a single unit. Maybe the consumer is doing too much? Could the consumer be separated from the service so the consumer only worries about consuming from Kafka and the service handles the processing?